src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Notifications/Components/index.tsx

```diff
@@ -22,7 +22,9 @@ export const Notifications = withErrorBoundary(
       <S.LeftContainer>
         {picture && <S.Image src={picture} />}
         {textLeft}
-        <S.Text color={titleColor}>&nbsp; {title} &nbsp;</S.Text>
+        <S.Text color={titleColor}>
+          {'\u00A0'} {title} {'\u00A0'}
```

Comment:
Curious to know why you replaced &nbsp;?

Reply:
Was facing some issue with &nbps; :sweat_smile: , seems like &nbsp; and ' ' are not totally similar. https://stackoverflow.com/questions/1357078/whats-the-difference-between-nbsp-and#:~:text=The%20entity%20produces%20a,has%20the%20character%20code%20160.